pkg/kubectl/resource_printer.go

```diff
@@ -786,8 +788,17 @@ func printPodBase(pod *api.Pod, w io.Writer, options PrintOptions) error {
 			return err
 		}
 	}
+
+	podColor := "<fg 2>" // green
+	if readyContainers != totalContainers {
+		podColor = "<fg 11>" // yellow
+	}
+	if readyContainers == 0 {
+		podColor = "<fg 1>" // red
```

Comment:
Still haven't seen this one addressed. Additionally if I add watch flag I'm getting:
```
$ kubectl get pods --color -w
NAME          READY     STATUS    RESTARTS   AGE
<fg 2>hello-5b209<reset>   1/1       Running   0          18s
```

and that's definitely not what I'd expect.